pkg/agent/cniserver/server.go

```diff
@@ -117,6 +117,7 @@ type CNIServer struct {
 	enableSecondaryNetworkIPAM bool
 	disableTXChecksumOffload   bool
 	secondaryNetworkEnabled    bool
+	multiClusterEnabled        bool
```

Comment:
I am thinking should CNIServer knows about multi-cluster at all, or we just need to pass a `mtuDeduction` (or `mtuDeductionPolicyOnly to be clear. But you may check how to change MTU for noEncap mode to decide) argument to it?

Reply:
Done